packages/salesforcedx-sobjects-faux-generator/test/fauxClassGenerator.test.ts

```diff
@@ -24,6 +24,43 @@ describe('SObject faux class generator', function() {
     }
   });
 
+  it('Should generate a faux class with a proper header comment', async () => {
+    const fieldsHeader = '{ "name": "Custom__c", "fields": [ ';
+    const closeHeader = ' ], "childRelationships": [] }';
+
+    const sobject1 = `${fieldsHeader}${closeHeader}`;
+
+    const sobjectFolder = process.cwd();
+    const gen = getGenerator();
+    classPath = await gen.generateFauxClass(
+      sobjectFolder,
+      JSON.parse(sobject1)
+    );
+    expect(fs.existsSync(classPath));
+    const classText = fs.readFileSync(classPath, 'utf8');
+    expect(classText).to.include(
+      '// This file is generated as an Apex representation'
```

Comment:
You should reference the message variable's value directly rather than a random substring here.

Reply:
yes, probably cleaner - I was checking for just a subset, but safer to use what is in the messages file - especially now that we are talking about making it internationalizable.  Will do